src/cascadia/TerminalApp/defaults.json

```diff
@@ -256,8 +273,8 @@
         { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" },
 
         // OS Integration

```

Comment:
Curious, what other future keybindings would fall under OS integration? The section title seems a bit broad for just copy/paste. Maybe it doesn't need its own comment? But then maybe that might look a little weird bc everything else has a section title hmm

Reply:
@DHowett-MSFT added this comment in before, I think

I'd be ok with "Clipboard Integration". I feel like Clipboard brings a bit more attention to it too.

Reply:
Make it up as you go, change it to whatever feels right